### PR TITLE
Improve release-drafter settings

### PR DESCRIPTION
**Pull request by ewjoachim**
Cf #146 

### Successful PR Checklist:

- [ ] Tests
  - [x] (not applicable?)
- [ ] Documentation
  - [x] (not applicable?)
- [x] Had a good time contributing?

### CODE REVIEW

File: .github/release-drafter.yml [hunk 1]
```diff
@@ -1,27 +1,31 @@
...
-    label: 'PR:major'
+    label: 'PR scope: major'
   minor:
-    label: 'PR:minor'
+    label: 'PR scope: minor'
   patch:
-    label: 'PR:incremental'
+    label: 'PR scope: incremental'
 exclude-labels:
 - 'PR:skip-changelog'
```
Comment (elemoine) [inline .github/release-drafter.yml:22]: I don't see this label in the list of labels. Don't you want to create it?

Comment (ewjoachim) [reply .github/release-drafter.yml:22]: Screenshot from 2020-06-11 10-03-43

Comment (ewjoachim) [reply .github/release-drafter.yml:22]: Screenshot from 2020-06-11 10-06-34

Comment (ewjoachim) [reply .github/release-drafter.yml:22]: AAAAaaaaaah your comment was pointing at the line BELOW the diff :o

Ok !

Comment (elemoine) [reply .github/release-drafter.yml:22]: Yes it was about `PR:skip-changelog`. Sorry for the confusion. And I am not sure I can create labels myself.

Comment (ewjoachim) [reply .github/release-drafter.yml]: I think you can. I've created it.

Screenshot from 2020-06-11 11-14-45

File: .github/release-drafter.yml [hunk 2]
```diff
@@ -1,27 +1,31 @@
...
 exclude-labels:
-- 'PR:skip-changelog'
+- 'PR: Skip changelog'
 template: |
-  ## Migrations:
+  ## Migrations
+
+  None (TODO: replace if there are migrations)
+
+  ## Uncategorized (TODO: please categorize)
```
Comment (elemoine) [inline .github/release-drafter.yml:28]: Aren't the changes output by `$CHANGES` supposed to be categorized?

**Pull request merged**